### ISSUE TITLE
min-free-space: The value '4G' doesn't match the pattern '/^[0-9]+[KMG]$/'.

### ISSUE THREAD

**Issue by joakimlemb** [OpenMediaVault-Plugin-Developers/openmediavault-unionfilesystems#28]
The regex pattern check is failing when trying to create a new mergerfs mount with OMV 4.1.7:

```
min-free-space: The value '4G' doesn't match the pattern '/^[0-9]+[KMG]$/'.
Error #0:
OMV\Json\SchemaValidationException: min-free-space: The value '4G' doesn't match the pattern '/^[0-9]+[KMG]$/'. in /usr/share/php/openmediavault/json/schema.inc:481
Stack trace:
#0 /usr/share/php/openmediavault/json/schema.inc(370): OMV\Json\Schema->checkPattern('4G', Array, 'min-free-space')
#1 /usr/share/php/openmediavault/json/schema.inc(297): OMV\Json\Schema->validateString('4G', Array, 'min-free-space')
#2 /usr/share/php/openmediavault/json/schema.inc(627): OMV\Json\Schema->validateType('4G', Array, 'min-free-space')
#3 /usr/share/php/openmediavault/json/schema.inc(395): OMV\Json\Schema->checkProperties(Object(stdClass), Array, '')
#4 /usr/share/php/openmediavault/json/schema.inc(285): OMV\Json\Schema->validateObject(Object(stdClass), Array, '')
#5 /usr/share/php/openmediavault/json/schema.inc(257): OMV\Json\Schema->validateType(Object(stdClass), Array, '')
#6 /usr/share/php/openmediavault/rpc/paramsvalidator.inc(59): OMV\Json\Schema->validate(Object(stdClass))
#7 /usr/share/php/openmediavault/rpc/serviceabstract.inc(178): OMV\Rpc\ParamsValidator->validate('{"name":"merger...')
#8 /usr/share/openmediavault/engined/rpc/unionfilesystems.inc(182): OMV\Rpc\ServiceAbstract->validateMethodParams(Array, 'rpc.unionfilesy...')
#9 [internal function]: OMV\Engined\Rpc\UnionFilesystems->set(Array, Array)
#10 /usr/share/php/openmediavault/rpc/serviceabstract.inc(123): call_user_func_array(Array, Array)
#11 /usr/share/php/openmediavault/rpc/rpc.inc(86): OMV\Rpc\ServiceAbstract->callMethod('set', Array, Array)
#12 /usr/sbin/omv-engined(536): OMV\Rpc\Rpc::call('UnionFilesystem...', 'set', Array, Array, 1)
#13 {main}
```

**Comment by ryecoaaron** [comment]
This is caused by the changes in 4.1.7. I will have to fix this.

**Comment by ryecoaaron** [comment]
Fixed in 4.0.2 in the repo now.

**Comment by joakimlemb** [comment]
@ryecoaaron That was fast, thanks.
